quick-start/examples/n8.min.js

```diff
@@ -0,0 +1,36 @@
+import { Scene, Circle, Rectangle, Color, RadialGradient }
+    from "https://unpkg.com/pencil.js/dist/pencil.esm.js";
+
+const scene = new Scene(undefined, {
+    fill: new Color(),
+});
+const { width, height } = scene;
+scene.cursorPosition.set(scene.center).subtract(0, height / 4);
+
+const sun = new Circle(scene.center.add(0, height / 2), 100, {
+    fill: "orange",
+});
+
+const ocean = new Rectangle([0, height / 2], width, height / 2, {
+    // Use a radial-gradient as fill option
```

Comment:
```suggestion
    // Use a radial gradient as fill option
```
It's a compound noun which commonly use spaces. CSS uses `radial-gradient()` with a hyphen because a space would indicate the next token to the CSS parser.